docs/source/store-and-retrieve/retrievers.md

```diff
@@ -62,12 +62,12 @@ retrievers:
 ```
 In this example the `uri`, `collection_name`, and `top_k` are specified, while the default values for `output_fields` and `timeout` are used, and the `nvidia_api_key` will be pulled from the `NVIDIA_API_KEY` environment variable.
 
-This configured retriever can then be used as an argument for a function which uses a retriever (such as the `aiq_retriever` function). The `aiq_retriever` function is a simple function to provide the configured retriever as an LLM tool. Its config is shown below
+This configured retriever can then be used as an argument for a function which uses a retriever (such as the `retriever_tool` function). The `retriever_tool` function is a simple function to provide the configured retriever as an LLM tool. Its config is shown below
 
 ```python
-class AIQRetrieverConfig(FunctionBaseConfig, name="aiq_retriever"):
+class RetrieverConfig(FunctionBaseConfig, name="retriever_tool"):
```

Comment:
This should match the class definition in `src/nat/tool/retriever.py` (or update the code to match this).
```suggestion
class RetrieverConfig(FunctionBaseConfig, name="nat_retriever"):
```